src/components/Property/types.ts

```diff
@@ -1,6 +1,18 @@
 // These props are the same for all property types
-export interface ConcretePropertyBaseProps {
-  name: string;
-  description: string;
-  disabled: boolean;
+export interface PropertyProps {
+  uri: string;
+}
+
+export interface AdditionalDataNumber {
+  Exponent: number;
+  MaximumValue: number;
+  MinimumValue: number;
+  SteppingValue: number;
+}
+
+export interface AdditionalDataVectorMatrix {
+  Exponent: number; // TODO: handle the exponent
+  MaximumValue: number[];
+  MinimumValue: number[];
+  SteppingValue: number[];
 }
```

Comment:
Add all the other AdditionalData types as well! 

However, I'm wondering if this file is needed or if it would make sense to have the additionaldata struct in the file for the specific property type?